### PR TITLE
fixed Dolphin pics for AE26Ax, removed bad pics for AE269x

### PR DESCRIPTION
**Pull request by dziban303**
## Describe your changes

USCG Dolphins:
* Corrected pics where possible on hex `AE26Ax`, otherwise, removed bad pics
* Deleted bad pics on hex `AE269x`

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Comment by dziban303** [comment]
Done for now

**Review by Phaeton — CHANGES_REQUESTED** [review]
Would you update to use `/full/` versus `/400/`? I spot checked the photos and they look correct.

File: plane_images.csv [hunk 1]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
-AE26AA,https://cdn.jetphotos.com/full/6/14450_1639026032.jpg,https://cdn.jetphotos.com/full/5/51279_1636766293.jpg,https://cdn.jetphotos.com/full/6/55743_1634485851.jpg,
-AE26A9,https://cdn.jetphotos.com/full/6/74810_1641059578.jpg,https://cdn.jetphotos.com/full/6/74810_1641059578.jpg,https://cdn.jetphotos.com/full/5/39058_1642186393.jpg,
-AE26A8,https://cdn.jetphotos.com/full/6/58414_1634994324.jpg,https://cdn.jetphotos.com/full/6/47525_1611481661.jpg,https://cdn.jetphotos.com/full/6/18011_1608027875.jpg,
-AE26A7,https://cdn.jetphotos.com/full/5/76844_1642073701.jpg,https://cdn.jetphotos.com/full/6/97465_1641430090.jpg,https://cdn.jetphotos.com/full/6/84727_1642323336.jpg,
-AE26A6,https://cdn.jetphotos.com/full/5/37842_1641861301.jpg,https://cdn.jetphotos.com/full/5/80998_1641658080.jpg,https://cdn.jetphotos.com/full/6/24027_1642549659.jpg,
+AE26AF,,,,
+AE26AE,https://cdn.jetphotos.com/400/5/25018_1668386683.jpg,https://cdn.jetphotos.com/400/5/14226_1536279617.jpg,https://cdn.jetphotos.com/400/2/75250_1162534302.jpg,https://cdn.jetphotos.com/400/1/17801_1162916208.jpg
```
Comment (Phaeton) [inline plane_images.csv:1126]:
```suggestion
AE26AE,https://cdn.jetphotos.com/full/5/25018_1668386683.jpg,https://cdn.jetphotos.com/full/5/14226_1536279617.jpg,https://cdn.jetphotos.com/full/2/75250_1162534302.jpg,https://cdn.jetphotos.com/full/1/17801_1162916208.jpg
```

File: plane_images.csv [hunk 2]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26AD,,,,
+AE26AC,https://cdn.jetphotos.com/400/5/1282233_1690414286.jpg,https://cdn.jetphotos.com/400/6/25466_1580460131.jpg,https://cdn.jetphotos.com/400/5/40798_1559415986.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1128]:
```suggestion
AE26AC,https://cdn.jetphotos.com/full/5/1282233_1690414286.jpg,https://cdn.jetphotos.com/full/6/25466_1580460131.jpg,https://cdn.jetphotos.com/full/5/40798_1559415986.jpg,
```

File: plane_images.csv [hunk 3]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26AB,https://cdn.jetphotos.com/400/5/92669_1480542071.jpg,https://cdn.jetphotos.com/400/2/90389_1295731700.jpg,https://cdn.jetphotos.com/400/1/93650_1227247081.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1129]:
```suggestion
AE26AB,https://cdn.jetphotos.com/full/5/92669_1480542071.jpg,https://cdn.jetphotos.com/full/2/90389_1295731700.jpg,https://cdn.jetphotos.com/full/1/93650_1227247081.jpg,
```

File: plane_images.csv [hunk 4]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26AA,https://cdn.jetphotos.com/full/5/51279_1636766293.jpg,https://cdn.jetphotos.com/400/5/56498_1568028459.jpg,https://cdn.jetphotos.com/400/6/99107_1568200328.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1130]:
```suggestion
AE26AA,https://cdn.jetphotos.com/full/5/51279_1636766293.jpg,https://cdn.jetphotos.com/full/5/56498_1568028459.jpg,https://cdn.jetphotos.com/full/6/99107_1568200328.jpg,
```

File: plane_images.csv [hunk 5]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A9,https://cdn.jetphotos.com/full/6/82306_1447207621.jpg,,,
+AE26A8,https://cdn.jetphotos.com/400/6/1281268_1681701698.jpg,https://cdn.jetphotos.com/400/3/95180_1361412089.jpg,https://cdn.jetphotos.com/400/2/66029_1220408566.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1132]:
```suggestion
AE26A8,https://cdn.jetphotos.com/full/6/1281268_1681701698.jpg,https://cdn.jetphotos.com/full/3/95180_1361412089.jpg,https://cdn.jetphotos.com/full/2/66029_1220408566.jpg,
```

File: plane_images.csv [hunk 6]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A7,https://cdn.jetphotos.com/400/6/79446_1555524694.jpg,https://cdn.jetphotos.com/400/6/64360_1540260466.jpg,https://cdn.jetphotos.com/400/2/84840_1211314794.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1133]:
```suggestion
AE26A7,https://cdn.jetphotos.com/full/6/79446_1555524694.jpg,https://cdn.jetphotos.com/full/6/64360_1540260466.jpg,https://cdn.jetphotos.com/full/2/84840_1211314794.jpg,
```

File: plane_images.csv [hunk 7]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A6,https://cdn.jetphotos.com/400/5/33696_1622707610.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:1134]:
```suggestion
AE26A6,https://cdn.jetphotos.com/full/5/33696_1622707610.jpg,,,
```

File: plane_images.csv [hunk 8]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
-AE26A1,https://cdn.jetphotos.com/full/6/66096_1639981382.jpg,https://cdn.jetphotos.com/full/6/66096_1639981382.jpg,https://cdn.jetphotos.com/full/5/70056_1638661524.jpg,
-AE26A0,https://cdn.jetphotos.com/full/6/58218_1641561983.jpg,https://cdn.jetphotos.com/full/6/27163_1641536473.jpg,https://cdn.jetphotos.com/full/5/18445_1641488721.jpg,
-AE269F,https://cdn.jetphotos.com/full/6/98779_1626267956.jpg,https://cdn.jetphotos.com/full/5/52700_1587473021.jpg,https://cdn.jetphotos.com/full/5/83900_1581834600.jpg,
-AE269E,https://cdn.jetphotos.com/full/6/11462_1640996550.jpg,https://cdn.jetphotos.com/full/5/81177_1638094080.jpg,https://cdn.jetphotos.com/full/6/51315_1633866656.jpg,
-AE269D,https://cdn.jetphotos.com/full/5/71403_1641400459.jpg,https://cdn.jetphotos.com/full/6/76657_1639432842.jpg,https://cdn.jetphotos.com/full/6/34324_1639039426.jpg,
-AE269C,https://cdn.jetphotos.com/full/5/18754_1633311803.jpg,https://cdn.jetphotos.com/full/5/70968_1612967270.jpg,https://cdn.jetphotos.com/full/5/78064_1611061617.jpg,
+AE26A4,https://cdn.jetphotos.com/400/2/96746_1039819332.jpg,https://cdn.jetphotos.com/400/1/83530_1085492344.jpg,,
```
Comment (Phaeton) [inline plane_images.csv:1136]:
```suggestion
AE26A4,https://cdn.jetphotos.com/full/2/96746_1039819332.jpg,https://cdn.jetphotos.com/full/1/83530_1085492344.jpg,,
```

File: plane_images.csv [hunk 9]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A3,,,,
+AE26A2,https://cdn.jetphotos.com/400/6/19906_1586027811.jpg,https://cdn.jetphotos.com/400/2/34371_1187482693.jpg,,
```
Comment (Phaeton) [inline plane_images.csv:1138]:
```suggestion
AE26A2,https://cdn.jetphotos.com/full/6/19906_1586027811.jpg,https://cdn.jetphotos.com/full/2/34371_1187482693.jpg,,
```

File: plane_images.csv [hunk 10]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A1,https://cdn.jetphotos.com/400/5/34911_1604623866.jpg,https://cdn.jetphotos.com/400/3/63756_1317922352.jpg,https://cdn.jetphotos.com/400/2/27715_1310328358.jpg,
```
Comment (Phaeton) [inline plane_images.csv:1139]:
```suggestion
AE26A1,https://cdn.jetphotos.com/full/5/34911_1604623866.jpg,https://cdn.jetphotos.com/full/3/63756_1317922352.jpg,https://cdn.jetphotos.com/full/2/27715_1310328358.jpg,
```

File: plane_images.csv [hunk 11]
```diff
@@ -1122,38 +1122,38 @@ AE26B3,,,,
...
+AE26A0,https://cdn.jetphotos.com/400/2/20877_1299717505.jpg,,,
```
Comment (Phaeton) [inline plane_images.csv:1140]:
```suggestion
AE26A0,https://cdn.jetphotos.com/full/2/20877_1299717505.jpg,,,
```

**Review by Phaeton — CHANGES_REQUESTED** [review]
Would you update  `/400/` to `/full/`? I spot checked the photos and they look correct. :helicopter:

**Comment by dziban303** [comment]
Sorry, forgot to replace the URLs.

File: plane_images.csv (hunk 11 above)
Comment (dziban303) [reply plane_images.csv:1140]: done

**Review by Phaeton — APPROVED** [review]

**Pull request merged**